SQL/0000-00-00-schema.sql

```diff
@@ -1990,7 +2013,8 @@ CREATE TABLE `genomic_files` (
   `AnalysisModality` varchar(100),
   PRIMARY KEY (`GenomicFileID`),
   KEY `AnalysisModality` (`AnalysisModality`),
-  CONSTRAINT `genomic_files_ibfk_1` FOREIGN KEY (`AnalysisModality`) REFERENCES `genomic_analysis_modality_enum` (`analysis_modality`)
+  CONSTRAINT `genomic_files_ibfk_1` FOREIGN KEY (`AnalysisModality`) REFERENCES `genomic_analysis_modality_enum` (`analysis_modality`),
+  CONSTRAINT `FK_genomic_files_FileTypes` FOREIGN KEY (`FileType`) REFERENCES `FileTypes`(`type`)
```

Comment:
The content of the FileType is usualy the mime type of the uploaded file. Since 'text/csv' is not in the FileTypes table, this will break the genomic_files.
Please remove this foreign key. 
We should rename that column to MimeType I guess.